new-sdks/app/package.json

```diff
@@ -66,6 +66,7 @@
     },
     "dependencies": {
         "@babel/runtime-corejs2": "^7.3.1",
+        "@token-io/core": "^1.0.0-beta.0",
```

Comment:
How does core get updated to use beta.1? Is this done automatically?

Reply:
it actually doesn't need to update. since core is bundled into tpp/app, I'm adding it here so the flow type file can be accessed. this doesn't increase the size either since it won't be bundled at the application level.